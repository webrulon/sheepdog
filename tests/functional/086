#!/bin/bash

# Test object reclaim

. ./common

for i in 0 1 2; do
    _start_sheep $i
done
_wait_for_sheep 3
_cluster_format -c 2

$DOG vdi create test 16M -P
_node_info

for i in `seq 0 3`; do
    $DOG vdi snapshot test -s snap$i
    $DOG vdi write test $((4 * 1024 ** 2 * $i)) 512 < /dev/zero
    _node_info
done

for i in `seq 0 3`; do
    $DOG vdi delete test -s snap$i
    sleep 1
    _node_info
done

$DOG vdi delete test
sleep 1
_node_info
