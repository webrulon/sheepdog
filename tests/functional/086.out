QA output created by 086
using backend plain store
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	3	0	0	0	0	0
1	3	1	0	0	0	0
2	2	1	0	0	0	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	4	1	0	0	1	0
1	4	1	0	0	0	0
2	4	2	0	0	1	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	6	2	0	0	2	0
1	5	1	0	0	0	0
2	5	3	0	0	2	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	8	2	0	0	3	0
1	7	2	0	0	1	0
2	5	4	0	0	2	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	9	3	0	0	4	0
1	9	3	0	0	2	0
2	6	4	0	0	2	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	7	3	0	0	3	0
1	8	3	0	0	2	0
2	5	4	0	0	1	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	6	3	0	0	3	0
1	7	3	0	0	2	0
2	5	4	0	0	1	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	5	3	0	0	3	0
1	7	3	0	0	2	0
2	4	4	0	0	1	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	5	3	0	0	3	0
1	5	3	0	0	1	0
2	4	4	0	0	2	0
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	0	3	0	0	0	0
1	0	3	0	0	0	0
2	0	4	0	0	0	0
